components/ai_chat/content/browser/ai_chat_tab_helper.cc

```diff
@@ -267,18 +175,25 @@ GURL AIChatTabHelper::GetPageURL() const {
 void AIChatTabHelper::GetPageContent(GetPageContentCallback callback,
                                      std::string_view invalidation_token) {
   bool is_pdf = IsPdf(web_contents());
-  if (is_pdf && !is_pdf_a11y_info_loaded_) {
-    SetPendingGetContentCallback(std::move(callback));
-    // Manually check when pdf extraction requested so we don't always rely on
-    // a11y events to prevent stale callback. It can happens during background
-    // pdf tab loading or bug in upstream kPdfOCR that an empty page in pdf will
-    // cause PdfOcrHelper::AreAllPagesOcred() return false becasue it won't get
-    // Ocred but `remaining_page_count_` was set to total pdf page count. Hence
-    // status IDS_PDF_OCR_COMPLETED or IDS_PDF_OCR_NO_RESULT won't be set.
-    CheckPDFA11yTree();
-    return;
+  bool pdf_helper_not_available = false;
+  if (is_pdf) {
```

Comment:
This if block is empty when ENABLE_PDF is false.

Reply:
I will make !ENABLE_PDF(ex. Android) to fallback to print preview extraction.
It wasn't able to load pdf in it back then but now since it is able to do so.

Reply:
changed that in cb5497fa783fb329dfe8faaacf690e8073ac6017